### PR TITLE
COS OCP SC-2 main control

### PR DESCRIPTION
**Pull request by thamill**
set status 'complete'
This control is IS specific, both COS and OCP allow for the distinction between user functionality and administrative functionality.

### CODE REVIEW

**Comment by isimluk** [comment]
LGTM.

This seems legit to me, but I may be totally off. Does anyone know why this is hanging in here? 

Is more documentation needed? Or anything else? Thanks!

File: coreos-4/policies/SC-Systems_and_Communications_Protection/component.yaml [hunk 1]
```diff
@@ -20,18 +20,11 @@
...
-        The customer is responsible for separating user functionality
-        including user interface services) from information system management
-        functionality. A successful control response will indicate how user
-        and management functionality access is separated.
-
-        A complete control response is planned. Engineering progress can be
-        tracked via:
-
-        https://issues.redhat.com/browse/CMP-429
+        CoreOS separates user functionality and administration functionality with the use of
```
Comment (redhatrises) [inline coreos-4/policies/SC-Systems_and_Communications_Protection/component.yaml:26]: CoreOS should be RHCOS

File: coreos-4/policies/SC-Systems_and_Communications_Protection/component.yaml [hunk 2]
```diff
@@ -20,18 +20,11 @@
...
+        standard Linux user and group permission configuration.
```
Comment (redhatrises) [inline coreos-4/policies/SC-Systems_and_Communications_Protection/component.yaml:27]: Do we have any links to documentation? Also, isn't this done by default?

Comment (redhatrises) [reply coreos-4/policies/SC-Systems_and_Communications_Protection/component.yaml:27]: However, this is about application partitioning. How does rhcos do this?

File: openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml [hunk 3]
```diff
@@ -20,18 +20,13 @@
...
-        and management functionality access is separated.
-
-        A complete control response is planned. Engineering progress can be
-        tracked via:
-
-        https://issues.redhat.com/browse/CMP-429
+        Application partitioning is implemented within OpenShift configuration by
+        enforcement of Role Based Access Control (RBAC) objects. RBAC objects support
+        separation of user functionality from admin functionality per project, and
+        supports separation of project admin functionality from OpenShift admin functionality.
```
Comment (redhatrises) [inline openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml:29]: Need link to RBAC documentation, but what about namespaces, just using a container, how master/worker nodes are designed for this, etc.?

**Comment by redhatrises** [comment]
The comments in this PR haven't been addressed or resolved in a few months. This PR looks frozen. Closing. Please re-open this PR when the comments are addressed appropriately.

**Pull request closed without merging**